paymentsheet/src/main/java/com/stripe/android/paymentelement/EmbeddedPaymentElementKtx.kt

```diff
@@ -68,4 +56,19 @@ fun rememberEmbeddedPaymentElement(
             resultCallback = onResult,
         )
     }
+
+    val callbacks = remember(builder) {
```

Comment:
I'm a little worried about this change (probably nothing, but we should verify!)

Particularly for when process death happens.

I think we could completely work around these fears by changing how we pass `embeddedPaymentElement` into the callback, or we could test the changes with process death.

Let me know what direction you want to choose, and I'm happy to help!